hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java

```diff
@@ -176,13 +176,6 @@ public int getSize() {
     }
 
     public EvictionConfig setSize(int size) {
-        return setSizeInternal(size);
-    }
-
-    /**
-     * Package private method for {@link EvictionConfigAccessor}.
-     */
-    EvictionConfig setSizeInternal(int size) {
```

Comment:
Not needed anymore, since we work on the modifiable configuration, before we make it a read-only one.